test_regress/t/t_cover_line.out

```diff
@@ -49,18 +49,18 @@
         
  000010    always @ (posedge clk) begin
 +000010  point: comment=block
-%000000       if (cyc!=0) begin
+ 000010       if (cyc!=0) begin
```

Comment:
I believe the old "min" was correct.  You see here why - the implied else (when cyc==0) is not being hit.

Reply:
I view coverage as "how many times was this line hit", and I think that's how most people view it. In [this example](https://stackoverflow.com/questions/62936628/how-to-improve-the-branch-coverage-using-gcovr-tool), this is the case, as well as other examples I've found online. If people want to see the individual branch coverage, it's right there underneath it.

Reply:
Viewing it as the min may highlight the error more, but I think from a glance presents a logical inconsistency when you see an if statement with 10 line hits inside, yet 0 line hits on the if statement. And now that both this and .info files support branches, I think it makes sense to use max.

Reply:
The way my site uses it is any line with "%" needs more attention. If there are none, then coverage is done, there's nothing more to look at.  This isn't "line coverage" but rather "is this line of code completely covered in all respects". 

So the change breaks that usage.  I'm open to other ways to resolve this usage difference.

Suggest again splitting the pull so we can get the branch coverage info merged, then we can debate this other change, thanks.

Reply:
BTW the % is like the yellow lines meaning there's no branch coverage, shown here:

https://app.codecov.io/gh/verilator/verilator/blob/master/src%2FV3ActiveTop.cpp

(Using Verilator coverage as I don't have a verilog example I can point to.)

Reply:
Is the % there because of the ok() check? Maybe that can be based on the min so it's flagged as partial? But I think it's easiest to just have a flag for this aggregation function

Reply:
Yes.  Maybe use  - for completely covered (lcov green), % for uncovered (lcov red) and new ~ for line covered but branch (or user cover) uncovered?

Reply:
I added just ~ to minimize the conflict, and becuase -/+ is already used for point annotations, and it means the opposite (red)

Reply:
Ok, remember to update docs on this in exe_verilator_coverage.rst